src/main/java/org/jenkinsci/maven/plugins/hpi/RunMojo.java

```diff
@@ -540,6 +534,7 @@ public void configureWebApplication() throws Exception {
         userStore.addUser("bob", new Password("bob"), new String[] {"user", "male"});
         userStore.addUser("charlie", new Password("charlie"), new String[] {"user", "male"});
         getWebAppConfig().getSecurityHandler().setLoginService(hashLoginService);
+        finishConfigurationBeforeStart();
```

Comment:
The super class no longer provides two distinct hooks here, but to keep the diff small I retained the existing split between two methods and made the first vector into the second.